### PR TITLE
Added demo reference in model docs

### PR DESCRIPTION
**Pull request by Anna-Mironova**
Links to demos in which OMZ models can be used are added to the documentation for models

### CODE REVIEW

File: models/intel/human-pose-estimation-0001/README.md [hunk 1]
```diff
@@ -41,5 +41,14 @@ The net outputs are two blobs:
...
 2. Name: `Mconv7_stage2_L2`, shape: `1, 19, 32, 57` contains keypoint heatmaps.
 
+## Demo usage
+
+The model can be used in the following demos provided by the OpenVINO toolkit to show its capabilities:
+
+* [Human Pose Estimation C++ Demo](../../../demos/human_pose_estimation_demo/cpp/README.md)
+* [Human Pose Estimation Python\* Demo](../../../demos/human_pose_estimation_demo/python/README.md)
+* [Multi-Channel Human Pose Estimation C++ Demo](../../../demos/multi_channel_human_pose_estimation_demo/cpp/README.md)
+* [Single Human Pose Estimation Demo](../../../demos/single_human_pose_estimation_demo/python/README.md)
```
Comment (eaidova) [inline models/intel/human-pose-estimation-0001/README.md:51]: I do not see this model in supported list for single human pose estimation 
https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/single_human_pose_estimation_demo/python
https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/single_human_pose_estimation_demo/python/models.lst

File: models/intel/instance-segmentation-person-0007/README.md [hunk 2]
```diff
@@ -49,6 +49,12 @@ The expected channel order is `BGR`
 
 The OpenVINO [Training Extensions](https://github.com/openvinotoolkit/training_extensions/blob/misc/README.md) provide a [training pipeline](https://github.com/openvinotoolkit/training_extensions/blob/misc/models/instance_segmentation/model_templates/custom-instance-segmentation/readme.md), allowing to fine-tune the model on custom dataset.
 
+## Demo usage
+
+The model can be used in the following demos provided by the OpenVINO toolkit to show its capabilities:
+
+* [Background subtraction Python\* Demo](../../../demos/background_subtraction_demo/python/README.md)
```
Comment (eaidova) [inline models/intel/instance-segmentation-person-0007/README.md:56]: probably instance-segmentation demo alsi support this model

File: models/public/facenet-20180408-102900/README.md [hunk 3]
```diff
@@ -73,6 +73,12 @@ An example of using the Model Converter:
 omz_converter --name <model_name>
 ```
 
+## Demo usage
+
+The model can be used in the following demos provided by the OpenVINO toolkit to show its capabilities:
+
+* [Face Recognition Python\* Demo](../../../demos/face_recognition_demo/python/README.md)
```
Comment (eaidova) [inline models/public/facenet-20180408-102900/README.md:80]: probably smart classroom is also suitable

File: models/intel/action-recognition-0001/README.md [hunk 4]
```diff
@@ -61,5 +61,11 @@ An embedding image, name: `0`, shape: `1, 16, 512` in the format `B, T, C`, wher
 
 The model outputs a tensor with the shape `1, 400`, each row is a logits vector of performed actions.
 
+## Demo usage
+
+The model can be used in the following demos provided by the OpenVINO toolkit to show its capabilities:
```
Comment (vladimir-dudnik) [inline models/intel/action-recognition-0001/README.md:66]:
```suggestion
The model can be used in the following demos provided by the Open Model Zoo to show its capabilities:
```

Comment (vladimir-dudnik) [reply models/intel/action-recognition-0001/README.md:66]: suggest to change to Open Model Zoo who is actually provides the demos

**Review by vladimir-dudnik — APPROVED** [review]
we need to think regarding automated check for this (and potentially automated regeneration of this section in README)

cc @nignatovsky @Wovchena

**Comment by Anna-Mironova** [comment]
Jenkins please retry a build

File: models/intel/human-pose-estimation-0001/README.md (hunk 1 above)
Comment (Anna-Mironova) [reply models/intel/human-pose-estimation-0001/README.md]: Fixed

File: models/public/facenet-20180408-102900/README.md (hunk 3 above)
Comment (Anna-Mironova) [reply models/public/facenet-20180408-102900/README.md:80]: Added

File: models/intel/action-recognition-0001/README.md (hunk 4 above)
Comment (Anna-Mironova) [reply models/intel/action-recognition-0001/README.md]: Updated

**Review by eaidova — APPROVED** [review]
